build/pipelines/release.yml

```diff
@@ -89,6 +89,16 @@ jobs:
     displayName: Use NuGet 5.10
     inputs:
       versionSpec: 5.10
+  # In the Microsoft Azure DevOps tenant, NuGetCommand is ambiguous.
+  # This should be `task: NuGetCommand@2`
+  - task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2

```

Comment:
Why do you have to use the guid here but not in the task immediately under this?

Reply:
Uhhhh at some point the GUID was required. Somehow now the name is working. Filing a follow on to unify them as I don't want to poke the bear here.

Reply:
#11874